cfme-migration_analytics.gemspec

```diff
@@ -1,5 +1,7 @@
 $:.push File.expand_path("../lib", __FILE__)
 
+require "json"
+
```

Comment:
Its not good to require gems in your gemspec, you can add the dependency lower down like `s.add_dependency "json"` but this probably isn't required

Reply:
Good to know. It looks like it still works with this removed (so I guess MIQ plugins all share dependencies? i wasn't sure). So I'll just leave it out.

Reply:
Yeah but with this here I was getting errors running it locally because of a mixture of json 2.2 being loaded by this and json 2.1 being used by the rest of the app